settings.gradle

```diff
@@ -16,3 +16,4 @@ include 'gutil'
 include 'lazily-configured-mapping'
 include 'circleci-artifacts'
 include 'develocity-adapter-utils'
+include 'integration-test-utils'
```

Comment:
I could see this having multiple artifacts and living in it's own repo. This gradle-utils was meant for small utility methods that don't deserve a whole new repo. Making a new repo is pretty low overhead, and it would probably be good to avoid making another large repo with lots of stuff in like gradle-baseline.

Reply:
ya, was going back and forth on that, but agreed.